react/ConditionLayoutCategory.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from 'react'
+import type { ComponentType } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+import ConditionLayout from './ConditionLayout'
+import type { NoUndefinedField, MatchType, Condition, Handlers } from './types'
+
+type Props = {
+  conditions: Array<Condition<ContextValues, HandlerArguments>>
+  matchType?: MatchType
+  Else?: ComponentType
+  Then?: ComponentType
+}
+
+type ContextValues = {
+  id: string,
+  type: 'category' | 'department'
+}
+
+type HandlerArguments = {
+  categoryId: { ids: string[] }
+  departmentId: { ids: string[] }
+}
+
+const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
+  categoryId({ values, args }) {
+    if(values.type === 'category') {
+
+      return args.ids.includes(values.id)
+    }
+
+    return false; 
```

Comment:
We have a pattern here, that is, the `happy path` should be on the left as much as possible. So knowing that, what do you think about this suggestion?
```suggestion
    if(values.type !== 'category') {
      return false
    }

    return args.ids.includes(values.id)
```